app/main.py

```diff
@@ -1,3 +1,61 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: Any) -> "Distance":
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Any) -> "Distance":
+        return Distance(round(self.km / other, 2))
```

Comment:
In the `__truediv__` method, there is a risk of division by zero if `other` is zero. Add a check to handle this case and raise an appropriate exception or return a specific value.